### DISCUSSION TITLE
Prevent CSS Animation from stopping when WinForms window is minimized

### DISCUSSION THREAD

**Discussion by grimaldijoao** [cefsharp/CefSharp#4202] [Q&A] [answered]
Basically the question explains it all, when i click outside my form the animation keeps running correctly, but when i minimize the application it halts execution.

(You can reproduce this by capturing the WinForms window in OBS or Discord)

**Comment by amaitland** [comment, marked as answer]
>  but when i minimize the application it halts execution.

This is the expected behaviour. 

You can create your own class that inherits from ChromiumWebBrowser and override ResizeBrowser, ignore calls when size is 0,0

https://github.com/cefsharp/CefSharp/blob/cefsharp/104/CefSharp.WinForms/ChromiumWebBrowser.cs#L821

**Reply by grimaldijoao** [reply]
Awesome! Thanks for the answer
